src/pages/quiz/[slug].jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import Link from 'next/link';
+
+// Server-only code.
+const fs = require('fs')
+export async function getStaticPaths() {
+    const files = fs.readdirSync('quizzes');
+    const paths = files.map((fileName) => ({
+        params: {
+        slug: fileName.replace('.json', ''),
+        },
+    }));
+
+        
+    // return file path
+    return {
+        paths,
+        fallback: false,
+    };
+}
+
+export function getStaticProps({ params: { slug } }) {
+    // read the file questions from the quizzes folder
+    const questions = fs.readFileSync(`quizzes/${slug}.json`, 'utf8');
+
+    // return the questions data as props
+    return {
+        props: {
+            data: JSON.parse(questions),
+        }
+    }
+}
+
+const Quiz = (questions) => {
+    const [currentQuestion, setCurrentQuestion] = React.useState(0);
+    const [score, setScore] = React.useState(0);
+    const [showScore, setShowScore] = React.useState(false);
+
+    // function to handle every question 
+    const handleQuestion = (answer) => {
```

Comment:
I wonder could we add handler for this function to send a POST request when the users choose wrong question?

```
if (user.answer IS CORRECT) {
givethemscore(1)
} else {
POST https://api.db?quizid=UUID&questionID=UUID&timestamp=TIMESTAMP
}
```